README.md

```diff
@@ -1,4 +1,241 @@
-# orm-mapping-to-tabel
+# ORM: Mapping Ruby Classes to Database Tables
+
+## Objectives
+
+1. Understand the concept of mapping a Ruby class to a database table and an instance of a class to a table row.
+2. Learn how to write code that maps a Ruby class to a database table. 
+3. Learn how to write code that inserts data regarding an instance of a class into a database table row. 
+
+## Mapping a Class to a Table
+
+When building an ORM to connect our Ruby program to a database, we equate a class with a database table and the instances that the class produces to rows in that table. 
+
+Let's say we are building a music player app that allows users to store their music and browse their songs by song.
+
+This program will have a `Song` class. Each song instance will have a name and an album attribute. 
+
+```ruby
+class Song
+
+  attr_accessor :name, :album
+  
+  def initialize(name, album)
+    @name = name
+    @album = album
+  end
+
+end
+```
+
+Here we have an `attr_accessor` for `name` and `album`. In order to "map" this `Song` class to a songs database table, we need to create our database, then we need to create our songs table. 
```

Comment:
Mention that our end goal is really to persist our song information to a database. That's why we are mapping. So something like "in order to persist the data, we need to first create a mapping from object to table" I know we will have covered the why in a previous reading, but I think it's important to re-iterate